src/js/components/CopyToClipboard.js

```diff
@@ -26,20 +26,23 @@ export default class extends React.PureComponent {
     }
 
     handleCopy = () => {
-        const container = document.createElement('textarea');
         const { clickCallback, src, namespace } = this.props;
 
-        container.innerHTML = JSON.stringify(
+        const textToCopy = JSON.stringify(
             this.clipboardValue(src),
             null,
             '  '
         );
 
-        document.body.appendChild(container);
-        container.select();
-        document.execCommand('copy');
-
-        document.body.removeChild(container);
+        if (navigator.clipboard) {
+            await navigator.clipboard.writeText(textToCopy);
```

Comment:
test is failing because await

Reply:
sorry, i should have used VSCode instead of just GH edit 😉